tests/console/velociraptor_client_event_collect.pm

```diff
@@ -91,11 +85,12 @@ sub run {
 
     # install virtualenv
     assert_script_run "$system_python_version  -m venv vrr-api";
+    assert_script_run "vrr-api/bin/pip install pip --upgrade";
     assert_script_run "vrr-api/bin/pip install pyvelociraptor";
     # add client monitoring
     assert_script_run('vrr-api/bin/pyvelociraptor --config api.config.yaml \'SELECT add_client_monitoring(artifact="SUSE.Linux.Events.ExecutableFiles") FROM scope()\'');
-    sleep 90;
     # generetae client events
+    sleep 10;
```

Comment:
10 seconds is not enough here - it can take up to 2 min I think. The other settings we made only affect the time it takes for events to be sent to the server, but not how often the client checks or is notified to update its event table. I was not able to find settings for that. Maybe just restart the client here so it gets the new table quickly.

Reply:
test  checks that generated events exist in` /var/tmp/velociraptor/clients/C.e51a2cc271ea0913/monitoring/` and if yes it pass so far 10 seconds seem was ok.

Reply:
Found it  https://github.com/Velocidex/velociraptor/blob/f7724a67a4b55c2a8f0a2534913d8e1e222958f1/config/proto/config.proto#L1130-L1133
So we should do
```yq -i ".defaults.event_change_notify_all_clients = true" server.conf```